### ISSUE TITLE
Simplify public API

### ISSUE THREAD

**Issue by cuducos** [datasciencebr/serenata-toolbox#59]
IMHO our public API is quite redundant — surely redundancy can be good sometimes… so I ask does it worth it to simplify it in these terms?

Turns stuff like:

```python
from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
```

In:

```python
from serenata_toolbox.federal_senate.dataset import Dataset
```

Or, at least"

```python
from serenata_toolbox.federal_senate.dataset import FederalSenateDataset
```

(And the same for `serenata_toolbox.chamber_of_deputies`).

**Comment by anaschwendler** [comment]
So you think that we should work on it?
The only thing that I can say about that is that by doing that we will need to fix that in a few places, but it's ok to change that :)

@jtemporal thoughts?

**Comment by lipemorais** [comment]
If were are talking about breaking changes...

What do you all think about the idea of have just one method to have a dataset, instead of call 3 methods(`fetch`, `clean` , `translate`)? I think that it will make the use of toolbox simpler and harder to break the API again.

**Comment by anaschwendler** [comment]
For me it looks real great, me and @jtemporal could totally work on it. I will only wait for her approval and attempt to do that.

So we are stating that we are to refactor @cuducos and @lipemorais suggestions, what do you think @jtemporal?

**Comment by jtemporal** [comment]
Hi y'all, sorry it took me a while to get to this issue.

As far as the fix the redundancy I believe is a great thing to work now. I would tackle it first and then consider reorganizing `fetch`, `clean`, `translate`.

Does that sound like a plan?

**Comment by cuducos** [comment]
> What do you all think about the idea of have just one method to have a dataset, instead of call 3 methods(fetch, clean , translate)? 

I think it's a good idea, but it's **not** the focus of this issue. This issue was supposed to be a small refactor renaming some modules and classes — simple as that.

I reinforce that because I'm have been criticizing this whole public API put forward by the toolbox and I would like see a big refactor (as I said elsewhere, the `translate` method doesn't only translate etc.) and I think this is not the right place to handle it. I'll open an issue to discuss this overarching changes in the API.

**Comment by lipemorais** [comment]
@cuducos I agree with you.

We can use baby step and have enhancements delivered in small packages.

**Comment by anaschwendler** [comment]
@jtemporal @lipemorais @cuducos 
> This issue was supposed to be a small refactor renaming some modules and classes — simple as that.
So I will be working on that task today, prepare something for tomorrow, and then we check that big refactor. It is a small thing and then we can make the big refactor, as we did when introducing the Federal Senate to it.

**Comment by cuducos** [comment]
Closed by #89
